apps/state/config/config.exs

```diff
@@ -415,13 +427,22 @@ config :state, :stops_on_route,
       "21458",
       "9070206",
       "30203",
-      "4510"
+      "4510",
+      "9070503",
```

Comment:
why are these the bus stations vs the underlying stations since the shuttle stops are temporary?

Reply:
The shuttle bus stops above are technically _not_ part of their nearby/constituent stations, at least according to our data model, which follows GTFS best practice of only including stops as part of stations if they are enclosed within a building or otherwise off-street area.

A stop on the side of the general-traffic road, by this standard, isn't part of a station. Thus, it is these "random bus stops" are showing up as part of the Green Line.